source/integration_tests/test_signup.py

```diff
@@ -0,0 +1,18 @@
+"""Integration test for signup."""
+import json
+import pytest
+
+from funwithflags.definitions import SignupRequest
+from funwithflags.use_cases import signup
+
+
+@pytest.mark.usefixtures("context")
+def test_signup(context):
```

Comment:
Add more test cases to cover failure cases.

Reply:
This is relying on ad-hoc testing. But setting up automated system tests is a good idea. Will do in separate story.